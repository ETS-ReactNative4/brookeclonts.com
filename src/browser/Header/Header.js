import React, { Component } from 'react';
import ReactDOM from 'react-dom';
import PropTypes from 'prop-types';
import { 
    Link
  } from 'react-router-dom';
import { css } from 'emotion';
import Twitter from '../Icons/Twitter';
import Facebook from '../Icons/Facebook';
import Instagram from '../Icons/Instagram';
import { colors } from '../constants/colors.js';
import { breakpoints } from '../constants/breakpoints.js';

class Header extends Component {

    constructor(props) {
        super(props);

        this.state = {
            fixed: false,
            mobileMenuOpen: false,
        };
    }

    render() {
        const toggleMenu = () => {
            this.setState({mobileMenuOpen: !this.state.mobileMenuOpen})
        }

        const menuIcon = `                
            background: ${colors.blueGray};
            display: block;
            margin: 3px;
            height: 3px;
            width: 30px;
            border-radius: 12px;
            opacity: 1;
            transform: rotate(0deg);
            transform-origin: left center;
            transition: .25s ease-in-out;

            &:nth-of-type(1) {
                transform: ${this.state.mobileMenuOpen ? 'rotate(39deg)' : 'rotate(0deg)'};
            }
            &:nth-of-type(2) {
                width: ${this.state.mobileMenuOpen ? '0%' : '30px'};
                opacity: ${this.state.mobileMenuOpen ? 0 : 1};
            }
            &:nth-of-type(3) {
                transform: ${this.state.mobileMenuOpen ? 'rotate(-39deg)' : 'rotate(0deg)'};
            }
        `;

        const link = `
            color: $blue-gray;
            font-size: 20px;
            padding: 5px 20px;
        `;

        const mobileLinks = `
            background-color: $pink;
            color: $blue-gray;
            display: block;
            text-align: center;
            font-size: 16px;
            padding-top: 20px;
            padding-bottom: 30px;
            position: absolute;
            width: 100%;
            transition: all 0.5s ease;
            top: ${this.state.mobileMenuOpen ? '48px' : '-280px'};
            z-index: ${this.state.mobileMenuOpen ? 1 : 0};
        `;

        const mobileLink = `
            ${link}
            color: ${colors.blueGray};
            font-family: 'Open Sans',Helvetica,arial,sans-serif;
            text-decoration: none;
        `;

        const mobileLinkDiv = `
            padding: 10px 0;
        `;

        const desktopLink = `
            font-family: 'Open Sans', Helvetica, arial, sans-serif;
            color: ${colors.blueGray};
            font-size: 20px;
            padding: 5px 20px;
            text-decoration: none;
            ${link}
        `;

        return (
            <div
                className={css`
                    top: 0;
                    left: 0;
                    right: 0;
                    vertical-align: middle;
                    width: 100%;
                    position: ${this.state.fixed ? 'fixed' : 'absolute'}
            `   }>
                <div>
                    <div
                        className={css`
                            background: rgba(254, 255, 255, 0.85);
                            display: flex;
                            padding: 10px 0;
                            position: absolute;
                            width: 100%;
                            z-index: 5;
                        `}
                    >
                        <div 
                            className={css`
                                float: left;
                                font-family: 'Open Sans', Helvetica, arial, sans-serif;
                            `}
                        >
                            <Link 
                                className={css`
                                    color: ${colors.blueGray};
                                    font-family: 'Medula One', 'Times New Roman', serif;
                                    font-size: 26px;
                                    letter-spacing: 2px;
                                    text-transform: uppercase;
                                    text-decoration: none;
                                    padding-left: 20px;

                                    @media (min-width: ${breakpoints.tab}) {
                                        font-size: 38px;
                                    }
                                `}  
                                to="/"
                            >
                                brookeclonts.com
                            </Link>
                        </div>

                        <div 
                            className={css`
                                display: inline-flex;
                                margin-left: auto;
                                vertical-align: middle;
                                padding-right: 10px;

                                @media (max-width: ${breakpoints.tab}) {
                                    display: none;
                                }
                            `}  
                        >
                            <Link 
                                className={css`
                                    ${desktopLink}
                                `}
                                to="/books"
                            >
                                Books
                            </Link>
                            <Link 
                                className={css`
                                    ${desktopLink}
                                `}
                                to="/blog"
                            >
                                Blog
                            </Link>
                            <Link 
                                className={css`
                                    ${desktopLink}
                                `}
                                to="/about"
                            >
                                About
                            </Link>
                            <Link 
                                className={css`
                                    ${desktopLink}

                                    @media (max-width: ${breakpoints.desktop}) {
                                        display: none;
                                    }
                                `}
                                to="/projects"
                            >
                                Other Projects
                            </Link>
                            <div className=
                                {css`
                                    & Twitter, & Facebook, $ Instagram {
                                        vertical-align: top;
                                    }

                                    & svg {
                                        margin: 5px;
                                        vertical-align: middle;
                                    }

                                    & path {
                                        fill: ${colors.blueGray};
    
                                        &:hover {
                                            fill: ${colors.green};
                                        } 
                                    }
                                `}>
                                <a
                                    href="https://twitter.com/brookeclonts" 
                                    target="_blank"
                                >
                                    <Twitter size={'30px'}/>
                                </a>
                                <a 
                                    href="https://www.facebook.com/brookeclonts" 
                                    target="_blank"
                                >
                                    <Facebook size={'30px'}/>
                                </a>
                                <a 
                                    href="https://www.instagram.com/brookeclontsbooks/" 
                                    target="_blank"
                                >
                                    <Instagram size={'30px'}/>
                                </a>
                            </div>
                        </div>

                        <div 
                            className={css`
                                height: 28px;
                                width: 28px;
                                display: flex;
                                flex-direction: column;
                                justify-content: center;
                                align-items: center;
                                margin-left: auto;
                                padding-right: 20px;

                                &:hover {
                                    cursor: pointer;
                                }

                                @media (min-width: ${breakpoints.tab}) {
                                    display: none;
                                }
                            `}
                            onClick={toggleMenu}
                        >
                            <span 
                                className={css`
                                    ${menuIcon}            
                                `}
                            ></span>
                            <span 
                                className={css`
                                    ${menuIcon}            
                                `}
                            ></span>
                            <span 
                                className={css`
                                    ${menuIcon}            
                                `}
                            ></span>
                        </div>
                    </div>

                    <div 
                        className={css`
                            background-color: ${colors.pink};
                            color: ${colors.blueGray};
                            display: block;
                            text-align: center;
                            font-size: 16px;
                            padding-top: 20px;
                            padding-bottom: 30px;
                            position: absolute;
                            z-index: ${this.state.mobileMenuOpen ? 1 : 0};
                            width: 100%;
                            transition: all 0.5s ease;
                            top: ${this.state.mobileMenuOpen ? '48px' : '-300px'};

                            @media (min-width: ${breakpoints.tab}) {
                                display: none;
                            }
                        `}
                    >
                        <div 
                            className={css`
                                & svg {
                                    margin: 5px;
                                }

                                & svg path {
                                    fill: ${colors.blueGray}

                                &:hover {
                                    fill: ${colors.green};
                                } 
                                }
                            `}
                        >
                            <a href="https://twitter.com/brookeclonts" target="_blank">
                                <Twitter size={'30px'}/>
                            </a>
                            <a href="https://www.facebook.com/brookeclonts" target="_blank">
                                <Facebook size={'30px'}/>
                            </a>
                            <a href="https://www.instagram.com/brookeclontsbooks/" target="_blank">
                                <Instagram size={'30px'}/>
                            </a>
                        </div>
                        <div 
                            className={css`
                                ${mobileLinkDiv}
                            `}
                        >
                            <Link 
                                className={css`
                                    ${mobileLink}
                                `} 
                                to="/books" 
                                onClick={() => {this.scrollToBooks(this)}}
                            >
                                Books
                            </Link>
                        </div>
                        <div 
                            className={css`
                                ${mobileLinkDiv}
                            `}
                        >
                            <Link  
                                className={css`
                                    ${mobileLink}
                                `}
                                to="/blog"
                            >
                                Blog
                            </Link>
                        </div>
                        <div 
                            className={css`
                                ${mobileLinkDiv}
                            `}
                        >
                            <Link
                                className={css`
                                    ${mobileLink}
                                `} 
                                to="/about"
                            >
                                About
                            </Link>
                        </div>
                        <div 
                            className={css`
                                ${mobileLinkDiv}
                            `}
                        >
                            <Link 
                                className={css`
                                    ${mobileLink}
                                `}
                                to="/projects"
                            >
                                Other Projects
                            </Link>
                        </div>
                    </div>
                </div>
            </div>
        );
    }
}

export default Header;
