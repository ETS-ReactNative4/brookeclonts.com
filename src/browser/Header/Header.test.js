import React from 'react';
import ReactDOM from 'react-dom';
import Header from './Header';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';

const tree = renderer
        .create(
            <MemoryRouter>
                <Header/>
            </MemoryRouter>
        )
        .toJSON();

it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
});