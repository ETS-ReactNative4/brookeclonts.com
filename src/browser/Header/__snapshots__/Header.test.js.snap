// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-1axuzur"
>
  <div>
    <div
      className="css-dmgxk5"
    >
      <div
        className="css-1nrd48i"
      >
        <a
          className="css-17thjjc"
          href="/"
          onClick={[Function]}
        >
          brookeclonts.com
        </a>
      </div>
      <div
        className="css-4es31x"
      >
        <a
          className="css-1en3da9"
          href="/books"
          onClick={[Function]}
        >
          Books
        </a>
        <a
          className="css-1en3da9"
          href="/blog"
          onClick={[Function]}
        >
          Blog
        </a>
        <a
          className="css-1en3da9"
          href="/about"
          onClick={[Function]}
        >
          About
        </a>
        <a
          className="css-1tnmgg4"
          href="/projects"
          onClick={[Function]}
        >
          Other Projects
        </a>
        <div
          className="css-q06ao4"
        >
          <a
            href="https://twitter.com/brookeclonts"
            target="_blank"
          >
            <svg
              height="30px"
              version="1.1"
              viewBox="0 0 35 29"
              width="30px"
              xmlns="http://www.w3.org/2000/svg"
              xmlnsXlink="http://www.w3.org/1999/xlink"
            >
              <g
                fillRule="evenodd"
                id="Page-1"
                stroke="none"
                strokeWidth="1"
              >
                <g
                  fill="#FFFFFF"
                  id="Desktop-HD"
                  transform="translate(-699.000000, -11661.000000)"
                >
                  <g
                    id="footer"
                    transform="translate(-3.000000, 11446.000000)"
                  >
                    <path
                      d="M735.839992,215.536125 C734.459261,216.374227 732.933581,216.983756 731.302906,217.309891 C730.003941,215.888277 728.14748,215 726.092179,215 C722.152542,215 718.955304,218.278075 718.955304,222.320848 C718.955304,222.895069 719.016629,223.451636 719.139278,223.987761 C713.205665,223.682067 707.945692,220.77101 704.422319,216.336131 C703.807215,217.42139 703.455993,218.680401 703.455993,220.021178 C703.455993,222.559642 704.715933,224.800775 706.63279,226.113678 C705.463907,226.07837 704.360995,225.745731 703.396527,225.201243 L703.396527,225.290442 C703.396527,228.838903 705.857871,231.799205 709.126654,232.469129 C708.528275,232.641024 707.896447,232.727436 707.245106,232.727436 C706.785172,232.727436 706.336388,232.682836 705.901541,232.597353 C706.810259,235.504694 709.445356,237.622249 712.57012,237.678927 C710.127359,239.643171 707.047195,240.813912 703.70315,240.813912 C703.127071,240.813912 702.557496,240.780462 702,240.713563 C705.159143,242.787447 708.913877,244 712.944571,244 C726.078242,244 733.257858,232.846368 733.257858,223.171029 C733.257858,222.852328 733.252283,222.534555 733.240204,222.222358 C734.634872,221.190061 735.846496,219.901317 736.799814,218.433245 C735.521291,219.014899 734.144276,219.408862 732.700362,219.585403 C734.174938,218.680401 735.306655,217.245779 735.839992,215.536125"
                      fill={undefined}
                      id="Twitter"
                    />
                  </g>
                </g>
              </g>
            </svg>
          </a>
          <a
            href="https://www.facebook.com/brookeclonts"
            target="_blank"
          >
            <svg
              height="30px"
              version="1.1"
              viewBox="-1 0 30 30"
              width="30px"
              xmlns="http://www.w3.org/2000/svg"
              xmlnsXlink="http://www.w3.org/1999/xlink"
            >
              <path
                d="M0.605570082,0 C0.272093931,0 -0.000339128289,0.279555556 -0.000339128289,0.623175926 L-0.000339128289,29.0796019 C-0.000339128289,29.4232222 0.272093931,29.7027778 0.605570082,29.7027778 L15.2785207,29.7027778 L15.2785207,19.7773907 L12.181149,19.7773907 C11.9132377,19.7773907 11.6961956,19.5537463 11.6961956,19.27885 L11.6961956,15.2544148 C11.6961956,14.9795185 11.9132377,14.7558741 12.181149,14.7558741 L15.2785207,14.7558741 L15.2785207,11.9789556 C15.2785207,8.31910741 17.5133762,5.95453333 20.9747456,5.95453333 L24.0540304,5.96035741 C24.3219418,5.96035741 24.5389839,6.18400185 24.5389839,6.46006296 L24.5389839,10.1944593 C24.5389839,10.4716852 24.3219418,10.6953296 24.0540304,10.6953296 L21.9796957,10.6953296 C20.5485744,10.6953296 20.2874456,11.2695833 20.2874456,12.4006185 L20.2874456,14.7558741 L23.9093357,14.7558741 C24.0382045,14.7558741 24.1625515,14.8082907 24.2518553,14.9014759 C24.3422895,14.9958259 24.3942892,15.1227907 24.3942892,15.2544148 L24.3920283,19.27885 C24.3920283,19.5537463 24.1749862,19.7773907 23.9082053,19.7773907 L20.2863151,19.7773907 L20.2863151,29.7027778 L28.2207867,29.7027778 C28.5542628,29.7027778 28.8266959,29.424387 28.8266959,29.0807667 L28.8266959,0.623175926 C28.8266959,0.278390741 28.5565237,0 28.2207867,0 L0.605570082,0 Z"
                fill={undefined}
                fillRule="evenodd"
                id="Fill-1"
                stroke="none"
              />
            </svg>
          </a>
          <a
            href="https://www.instagram.com/brookeclonts"
            target="_blank"
          >
            <svg
              height="30px"
              version="1.1"
              viewBox="601 213 30 31"
              width="30px"
              xmlns="http://www.w3.org/2000/svg"
              xmlnsXlink="http://www.w3.org/1999/xlink"
            >
              <defs>
                <polygon
                  id="path-1"
                  points="602 243.938014 630.55272 243.938014 630.55272 214.000568 602 214.000568"
                />
              </defs>
              <g
                fill="none"
                id="Clip-2"
                stroke="none"
              />
              <path
                d="M627.459328,238.977948 C627.459328,239.924573 626.725567,240.693919 625.823811,240.693919 L606.729776,240.693919 C605.82802,240.693919 605.094259,239.924573 605.094259,238.977948 L605.094259,225.991115 L607.841797,225.991115 C607.531818,226.942286 607.325889,227.943458 607.325889,229.003724 C607.325889,234.176638 611.341521,238.388154 616.276251,238.388154 C621.212065,238.388154 625.226614,234.176638 625.226614,229.003724 C625.226614,227.943458 625.021768,226.942286 624.711789,225.991115 L627.459328,225.991115 L627.459328,238.977948 Z M616.276251,222.862593 C619.506099,222.862593 622.133331,225.618374 622.133331,229.003724 C622.133331,232.389074 619.506099,235.143719 616.276251,235.143719 C613.047488,235.143719 610.420256,232.389074 610.420256,229.003724 C610.420256,225.618374 613.047488,222.862593 616.276251,222.862593 L616.276251,222.862593 Z M622.702347,222.052336 L627.232805,222.052336 L627.232805,217.302165 L622.702347,217.302165 L622.702347,222.052336 Z M606.729776,213.999773 C604.122053,213.999773 601.999892,216.224853 601.999892,218.957906 L601.999892,238.977948 C601.999892,241.713274 604.122053,243.938354 606.729776,243.938354 L625.823811,243.938354 C628.431533,243.938354 630.553695,241.713274 630.553695,238.977948 L630.553695,218.957906 C630.553695,216.224853 628.431533,213.999773 625.823811,213.999773 L606.729776,213.999773 Z"
                fill={undefined}
                fillRule="evenodd"
                id="Fill-1"
                stroke="none"
              />
            </svg>
          </a>
        </div>
      </div>
      <div
        className="css-1aretml"
        onClick={[Function]}
      >
        <span
          className="css-1suncsr"
        />
        <span
          className="css-1suncsr"
        />
        <span
          className="css-1suncsr"
        />
      </div>
    </div>
    <div
      className="css-gempyx"
    >
      <div
        className="css-o3fx5z"
      >
        <a
          href="https://twitter.com/brookeclonts"
          target="_blank"
        >
          <svg
            height="30px"
            version="1.1"
            viewBox="0 0 35 29"
            width="30px"
            xmlns="http://www.w3.org/2000/svg"
            xmlnsXlink="http://www.w3.org/1999/xlink"
          >
            <g
              fillRule="evenodd"
              id="Page-1"
              stroke="none"
              strokeWidth="1"
            >
              <g
                fill="#FFFFFF"
                id="Desktop-HD"
                transform="translate(-699.000000, -11661.000000)"
              >
                <g
                  id="footer"
                  transform="translate(-3.000000, 11446.000000)"
                >
                  <path
                    d="M735.839992,215.536125 C734.459261,216.374227 732.933581,216.983756 731.302906,217.309891 C730.003941,215.888277 728.14748,215 726.092179,215 C722.152542,215 718.955304,218.278075 718.955304,222.320848 C718.955304,222.895069 719.016629,223.451636 719.139278,223.987761 C713.205665,223.682067 707.945692,220.77101 704.422319,216.336131 C703.807215,217.42139 703.455993,218.680401 703.455993,220.021178 C703.455993,222.559642 704.715933,224.800775 706.63279,226.113678 C705.463907,226.07837 704.360995,225.745731 703.396527,225.201243 L703.396527,225.290442 C703.396527,228.838903 705.857871,231.799205 709.126654,232.469129 C708.528275,232.641024 707.896447,232.727436 707.245106,232.727436 C706.785172,232.727436 706.336388,232.682836 705.901541,232.597353 C706.810259,235.504694 709.445356,237.622249 712.57012,237.678927 C710.127359,239.643171 707.047195,240.813912 703.70315,240.813912 C703.127071,240.813912 702.557496,240.780462 702,240.713563 C705.159143,242.787447 708.913877,244 712.944571,244 C726.078242,244 733.257858,232.846368 733.257858,223.171029 C733.257858,222.852328 733.252283,222.534555 733.240204,222.222358 C734.634872,221.190061 735.846496,219.901317 736.799814,218.433245 C735.521291,219.014899 734.144276,219.408862 732.700362,219.585403 C734.174938,218.680401 735.306655,217.245779 735.839992,215.536125"
                    fill={undefined}
                    id="Twitter"
                  />
                </g>
              </g>
            </g>
          </svg>
        </a>
        <a
          href="https://www.facebook.com/brookeclonts"
          target="_blank"
        >
          <svg
            height="30px"
            version="1.1"
            viewBox="-1 0 30 30"
            width="30px"
            xmlns="http://www.w3.org/2000/svg"
            xmlnsXlink="http://www.w3.org/1999/xlink"
          >
            <path
              d="M0.605570082,0 C0.272093931,0 -0.000339128289,0.279555556 -0.000339128289,0.623175926 L-0.000339128289,29.0796019 C-0.000339128289,29.4232222 0.272093931,29.7027778 0.605570082,29.7027778 L15.2785207,29.7027778 L15.2785207,19.7773907 L12.181149,19.7773907 C11.9132377,19.7773907 11.6961956,19.5537463 11.6961956,19.27885 L11.6961956,15.2544148 C11.6961956,14.9795185 11.9132377,14.7558741 12.181149,14.7558741 L15.2785207,14.7558741 L15.2785207,11.9789556 C15.2785207,8.31910741 17.5133762,5.95453333 20.9747456,5.95453333 L24.0540304,5.96035741 C24.3219418,5.96035741 24.5389839,6.18400185 24.5389839,6.46006296 L24.5389839,10.1944593 C24.5389839,10.4716852 24.3219418,10.6953296 24.0540304,10.6953296 L21.9796957,10.6953296 C20.5485744,10.6953296 20.2874456,11.2695833 20.2874456,12.4006185 L20.2874456,14.7558741 L23.9093357,14.7558741 C24.0382045,14.7558741 24.1625515,14.8082907 24.2518553,14.9014759 C24.3422895,14.9958259 24.3942892,15.1227907 24.3942892,15.2544148 L24.3920283,19.27885 C24.3920283,19.5537463 24.1749862,19.7773907 23.9082053,19.7773907 L20.2863151,19.7773907 L20.2863151,29.7027778 L28.2207867,29.7027778 C28.5542628,29.7027778 28.8266959,29.424387 28.8266959,29.0807667 L28.8266959,0.623175926 C28.8266959,0.278390741 28.5565237,0 28.2207867,0 L0.605570082,0 Z"
              fill={undefined}
              fillRule="evenodd"
              id="Fill-1"
              stroke="none"
            />
          </svg>
        </a>
        <a
          href="https://www.instagram.com/brookeclonts/"
          target="_blank"
        >
          <svg
            height="30px"
            version="1.1"
            viewBox="601 213 30 31"
            width="30px"
            xmlns="http://www.w3.org/2000/svg"
            xmlnsXlink="http://www.w3.org/1999/xlink"
          >
            <defs>
              <polygon
                id="path-1"
                points="602 243.938014 630.55272 243.938014 630.55272 214.000568 602 214.000568"
              />
            </defs>
            <g
              fill="none"
              id="Clip-2"
              stroke="none"
            />
            <path
              d="M627.459328,238.977948 C627.459328,239.924573 626.725567,240.693919 625.823811,240.693919 L606.729776,240.693919 C605.82802,240.693919 605.094259,239.924573 605.094259,238.977948 L605.094259,225.991115 L607.841797,225.991115 C607.531818,226.942286 607.325889,227.943458 607.325889,229.003724 C607.325889,234.176638 611.341521,238.388154 616.276251,238.388154 C621.212065,238.388154 625.226614,234.176638 625.226614,229.003724 C625.226614,227.943458 625.021768,226.942286 624.711789,225.991115 L627.459328,225.991115 L627.459328,238.977948 Z M616.276251,222.862593 C619.506099,222.862593 622.133331,225.618374 622.133331,229.003724 C622.133331,232.389074 619.506099,235.143719 616.276251,235.143719 C613.047488,235.143719 610.420256,232.389074 610.420256,229.003724 C610.420256,225.618374 613.047488,222.862593 616.276251,222.862593 L616.276251,222.862593 Z M622.702347,222.052336 L627.232805,222.052336 L627.232805,217.302165 L622.702347,217.302165 L622.702347,222.052336 Z M606.729776,213.999773 C604.122053,213.999773 601.999892,216.224853 601.999892,218.957906 L601.999892,238.977948 C601.999892,241.713274 604.122053,243.938354 606.729776,243.938354 L625.823811,243.938354 C628.431533,243.938354 630.553695,241.713274 630.553695,238.977948 L630.553695,218.957906 C630.553695,216.224853 628.431533,213.999773 625.823811,213.999773 L606.729776,213.999773 Z"
              fill={undefined}
              fillRule="evenodd"
              id="Fill-1"
              stroke="none"
            />
          </svg>
        </a>
      </div>
      <div
        className="css-1tzkzlf"
      >
        <a
          className="css-1nyyt18"
          href="/books"
          onClick={[Function]}
        >
          Books
        </a>
      </div>
      <div
        className="css-1tzkzlf"
      >
        <a
          className="css-1nyyt18"
          href="/blog"
          onClick={[Function]}
        >
          Blog
        </a>
      </div>
      <div
        className="css-1tzkzlf"
      >
        <a
          className="css-1nyyt18"
          href="/about"
          onClick={[Function]}
        >
          About
        </a>
      </div>
      <div
        className="css-1tzkzlf"
      >
        <a
          className="css-1nyyt18"
          href="/projects"
          onClick={[Function]}
        >
          Other Projects
        </a>
      </div>
    </div>
  </div>
</div>
`;
